### DISCUSSION TITLE
The NPM packages for players and editors

### DISCUSSION THREAD

**Discussion by rajeevsathish** [Sunbird-Knowlg/Community#139] [General Discussions]
Hi 
for the following packages 
@project-sunbird/sunbird-epub-player-v9 : "5.2.1",
@project-sunbird/sunbird-pdf-player-v9 : "5.1.1",
@project-sunbird/sunbird-video-player-v9 : "5.2.3",

in the release notes of knowlg (https://knowlg.sunbird.org/use/release-notes/release-5.5.0-latest) does not say what is the package as a adaptor we can take.
please suggest.
cc: @vinukumar-vs @Krishnaj20 
@ashokreddy1208, @amitpriyadarshi

**Comment by HarishGangula** [comment]
As part of this release, we have not done any functional or interface changes for the players only changes to support web component build are done.
